### DISCUSSION TITLE
Enable sphincssha2256ssimple

### DISCUSSION THREAD

**Discussion by lpantojaa** [open-quantum-safe/oqs-provider#437] [Q&A] [answered]
Hi!

I have been going crazy about this. I think that by default sphincssha2256ssimple is disabled. Is there a way to enable it?

**Comment by baentsch** [comment, marked as answer]
From the [README.md](https://github.com/open-quantum-safe/oqs-provider?tab=readme-ov-file#signature-algorithms):

> In addition, algorithms not denoted with "*" above are not enabled for TLS operations. This designation [can be changed by modifying the "enabled" flags in the main algorithm configuration file](https://github.com/open-quantum-safe/oqs-provider/blob/main/CONFIGURE.md#pre-build-configuration).

i.e., enable the alg in `generate.yml`, run `generate.py` and building as usual enables any supported algorithm.

**Reply by lpantojaa** [reply]
Thanks! I tried this before and I was missing the LIBOQS_SRC_DIR environment variable. I was able to add it now.
